src/index.html

```diff
@@ -1,12 +1,363 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8"></meta>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" ></meta>
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png"
+          class="logo__img"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active"> apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" data-qa="nav-hover"
+              class="nav__link"
+            > laptops & computers </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"> video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+
+      <div class="cards">
```

Comment:
```suggestion
    <main class="main">
      <div class="cards">
```
Don't leave newlines between parent and child elements